src/components/channels_list_view.js

```diff
@@ -57,7 +57,7 @@ class ChannelsListView extends Component {
                     </Button>
                 ))}
 
-                <ErrorText error={this.props.channels.error}/>
+                <ErrorText error={this.props.channel.error}/>
```

Comment:
Does it make sense to have an error control on every view?  Why not an error flyout control that listens to all error state similar to the webapp?  Since you don't care about listening for the actual error for async style calls?